openapi/tests/openapi_integration/test_init_from_collection.py

```diff
@@ -121,42 +124,36 @@ def create_multi_from_collection(collection_name, source_collection_name, vector
     return response
 
 
-def test_init_from_collection():
-    default_vector()
-    multi_vector()
-
-
-def default_vector():
-    # test successful init from collection
-    advanced_collection_setup(source_collection_name, 4, 'Dot', False)
-    response = create_from_collection(collection_name, source_collection_name, 4, 'Dot', False)
-    assert response.ok
-
-    # test failed init from collection (non existing source collection)
-    advanced_collection_setup(source_collection_name, 4, 'Dot', False)
-    response = create_from_collection(collection_name, "i-do-not-exist", 4, 'Dot', False)
-    assert not response.ok
-
-    # test failed init from collection (bad size)
-    advanced_collection_setup(source_collection_name, 4, 'Dot', False)
-    response = create_from_collection(collection_name, source_collection_name, 8, 'Dot', False)
-    assert not response.ok
-
-    # test failed init from collection (bad distance)
-    advanced_collection_setup(source_collection_name, 4, 'Dot', False)
-    response = create_from_collection(collection_name, source_collection_name, 4, 'Cosine', False)
-    assert not response.ok
-
-
-def multi_vector():
+@pytest.mark.parametrize("ok,source,size,distance", [
```

Comment:
:+1:

Reply:
There may be more places like this, but I didn't want to overdo it.